src/app/components/mno-app-install-btn/mno-app-install-btn.coffee

```diff
@@ -33,6 +33,39 @@ angular.module 'mnoEnterpriseAngular'
           else
             "INSTALLABLE"
 
+      vm.appInstallationBtnClick = ->
+        switch vm.appInstallationStatus()
+          when 'INSTALLABLE'
+            if vm.isExternallyProvisioned
+              vm.provisionOrder()
+            else
+              vm.provisionApp()
+          when 'INSTALLED_LAUNCH' then vm.launchAppInstance()
+          when 'INSTALLED_CONNECT' then vm.connectAppInstance()
+
+      vm.provisionConditions = ->
+        if vm.appInstance
+          vm.vm.appInstallationBtnClick()
```

Comment:
As @adamaziz15 pointed out, should there be two `vm`?

Reply:
No it should not. Fixed.